src/shared/ui/form/date/style.module.scss

```
@@ -1,16 +1,16 @@
   display: flex;
   flex-direction: column;
-  gap: token.$spacing_xs;
+  gap: token.$spacing_4;
 }
 
 .month_picker_label {
-  @include token.body_medium;
-  font-size: token.$font_size_sm;
-  color: token.$text_strong;
+  @include token.body_medium_medium;
+  font-size: token.$font_size_14;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)